### PR TITLE
A front-end realization of the minor improvement for the Jenkins Test Result page

### PR DESCRIPTION
**Pull request by aafedorov**
Please review testresult.md for details

### CODE REVIEW

**Comment by kevinburke** [comment]
Thanks for this! I don't have time to maintain this project anymore (and I don't use Jenkins) so I've given you push access. Once you feel you have this in a good place, feel free to merge. Thanks!

File: src/testresult.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * Adds a "Total:" row to the tables that contain test results, so the entire evolution can be tracked.
+ * @link https://issues.jenkins-ci.org/browse/JENKINS-35486
+ *
+ * @authors Andrei Fedorov (afedorov@luc.edu), Gabriel Borcean (gborcean@luc.edu)
+ *
+ * For details see testresult.md
+ */
+if ((window.location.pathname).includes("testReport")) {
+    // for debugging
+    console.log("Page found!");
```
Comment (kevinburke) [inline src/testresult.js]: Think we should remove this one

File: src/testresult.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                        break;
+                }
+            }
+        }
+
+        // for debugging
+        console.log(totalDuration, totalFail, totalSkip, totalPass, totalTotal);
+
+        // create total row and its cells
+        for (var i = 0; i < columns; i++) {
```
Comment (kevinburke) [inline src/testresult.js]: maybe pull this into a helper function - createTotalRow(newCells, columns) - and remove the comment?

File: src/testresult.js (hunk 1 above)
Comment (aafedorov) [inline src/testresult.js]: Agreed and updated

File: src/testresult.js (hunk 2 above)
Comment (aafedorov) [inline src/testresult.js]: Agreed and updated

**Comment by aafedorov** [comment]
Thank you for your replies and for letting me merge this!

**Pull request merged**